src/wireviz/DataClasses.py

```diff
@@ -47,13 +64,14 @@ def __post_init__(self, gv_dir):
                 if self.width:
                     self.height = self.width / aspect_ratio(gv_dir.joinpath(self.src))
 
+
 @dataclass
 class AdditionalComponent:
-    type: str
-    subtype: Optional[str] = None
-    manufacturer: Optional[str] = None
-    mpn: Optional[str] = None
-    pn: Optional[str] = None
+    type: MultilineHypertext
+    subtype: Optional[MultilineHypertext] = None
+    manufacturer: Optional[MultilineHypertext] = None
+    mpn: Optional[MultilineHypertext] = None
+    pn: Optional[Hypertext] = None
```

Comment:
Maybe the answer to my question is buried in the discussion on #115, but:

Why are `manufacturer` and `mpn` multiline-capable, and `pn` is not?

Personally, I'm not sure why any of them would need to be multiline, but at least it should be consistend, since all of them are passed through the `html_line_breaks()` function.

Reply:
> Maybe the answer to my question is buried in the discussion on #115, but:

This feature is much older that #115.

> 
> Why are `manufacturer` and `mpn` multiline-capable, and `pn` is not?

[Blame](https://github.com/formatc1702/WireViz/blame/dev/src/wireviz/Harness.py) tells me the change was committed by you in https://github.com/formatc1702/WireViz/commit/102c7d611377433c93b6227a8160c83106d594bd as suggested by me in https://github.com/formatc1702/WireViz/pull/136#discussion_r463942757: _In case a user need a long manufacturer info, I suggest supporting line breaks to avoid a very wide node._ And since both `manufacturer` and `mpn` was placed in the same table cell, I suggested calling `html_line_breaks` on the whole cell contents.

> 
> Personally, I'm not sure why any of them would need to be multiline, but at least it should be consistend, since all of them are passed through the `html_line_breaks()` function.

Currently, (since the commit mentioned above that is included in v0.2)  `manufacturer` and `mpn` are passed through the `html_line_breaks()` function, but not `pn`, and that's why I used different type hints to reflect how it is currently implemented.

I understand your argument about consistency, but then we also need to change the implementation. I have an idea about moving `html_line_breaks()` in #168 that will solve your consistency request, but it's still WIP.

Reply:
Thanks for digging through history for me :) 

OK, then let's keep it as-is to reflect the current implementation, and I'll wait for #168.